lib/utils.js

```diff
@@ -0,0 +1,35 @@
+function isI18nMethodCall(node, methodName) {
+    return node &&
+        node.type === 'CallExpression' &&
+        node.callee.type === 'MemberExpression' &&
+        node.callee.object.type === 'Identifier' &&
+        node.callee.object.name.toLowerCase() === 'i18n' &&
```

Comment:
So am I understanding correctly that the rules depend on the library being imported as `/i18n/i`? If I were to do `import internationalization from 'i18n';` then I would see no errors? Or `import i18n from 'other-library-with-a-gettext-method';` would see false positives? I realize theses are silly examples and there is probably no easy way around this, unless you can somehow identify the library explicitly to eslint; I just want to check my understanding.